### DISCUSSION TITLE
This is awsome and hugely impactful...

### DISCUSSION THREAD

**Discussion by tescophil** [t0bst4r/home-assistant-matter-hub#351] [Show and tell]
Just wanted to say thanks for this addon..., It's a game changer for Home Assistant and has allowed my system to become (more or less) cloud independent. I was relying on Nabu Casa for connections to Google and Alexa, as well as voice processing.

I've now moved to local voice processing, and with this addon now have local connections to both Google Home and Alexa.

I just stumbled upon this when browsing the community forums, it should be on the list of THE must have addons.

Thanks again for all your work on this :clap:

**Comment by pavichokche** [comment]
Yeah this is amazing! I just got into Home Assistant and this is so much better of a way to add my devices to Google Home than even SmartThings could do OR the horrible workaround way I would have needed to do using the google developer tools. It's so elegant and perfect...

**Comment by mucki12** [comment]
One more: Thanks!

This opens up completely new possibilities. I have now transferred almost 100 devices to Smartthings and can now operate them conveniently via the display on my refrigerator :-)
